test/unit/portal-target.spec.js

```diff
@@ -1,11 +1,12 @@
-import { expect, td } from './helpers'
```

Comment:
Please respect the code style. This repo has space paddings for inline object notation.

This applies to all changed files, I won't repeat it everywhere in the review for brevity.